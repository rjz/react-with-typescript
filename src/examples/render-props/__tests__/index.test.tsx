import * as React from 'react';
import * as TestRenderer from 'react-test-renderer';

import Counter from '../index';

describe('examples/render-props', () => {
  it('renders expected count', () => {
    const renderer = TestRenderer.create(<Counter />);
    expect(renderer.toJSON()).toMatchSnapshot();
  });
});
