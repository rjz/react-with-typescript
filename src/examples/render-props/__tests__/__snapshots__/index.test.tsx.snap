// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`examples/render-props renders expected count 1`] = `
<div>
  <h1>
    Count: 
    44
  </h1>
  <button
    onClick={[Function]}
  >
    +1
  </button>
  <button
    onClick={[Function]}
  >
    -1
  </button>
</div>
`;
